### PR TITLE
Write README.

### PR DESCRIPTION
**Pull request by josiah-tesfu**
Wrote a README for the client indicating it's use as a standalone service.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+Once installed, you can control the modules of the EdgePi directly through the SDK's wide functionality.
+
+For example, from a remote connection with address `localhost` and port `5555`, initialize a module as:
+
+```
+from edgepi_rpc_client.services.adc_service import ClientAdcService
+
+# initialize ADC Client
+adc_client = ClientAdcService('tcp://localhost:5555')
+```
```
Comment (naomiven) [inline README.md]: not a big deal, but you can add `python` after the first "```" to make it color coded and easier to read in the actual README:

```python
 from edgepi_rpc_client.services.adc_service import ClientAdcService

 # initialize ADC Client
 adc_client = ClientAdcService('tcp://localhost:5555')
 ```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,32 @@
-# edgepi-rpc-client-py
-EdgePi RPC Client Python
+![Image](https://user-images.githubusercontent.com/3793563/207438826-bb656ca5-f19d-4699-8cb4-35acccb2ce58.svg)
+
+# EdgePi RPC Client
+
+The RPC client exposes functionality of the EdgePi Python SDK over a network, allowing for remote control of the EdgePi.
+
+Since the client directly mirrors the SDK, you can take advantage of the user-friendly SDK functionality with the flexibility of a network connection.
```
Comment (farzadpanahi) [inline README.md]: it doesn't need to be over `network` it can use sockets too.

File: README.md (hunk 1 above)
Comment (farzadpanahi) [inline README.md]: this is not a full example

File: README.md [hunk 3]
```diff
@@ -1,2 +1,32 @@
...
+For example, from a remote connection with address `localhost` and port `5555`, initialize a module as:
+
+```
+from edgepi_rpc_client.services.adc_service import ClientAdcService
+
+# initialize ADC Client
+adc_client = ClientAdcService('tcp://localhost:5555')
+```
+
+From here, you can control the EdgePi exactly as you would through the SDK. Note that the RPC client module supports asynchronous configurations, and so multiple clients can be run concurrently.
```
Comment (farzadpanahi) [inline README.md]: > the RPC client module supports asynchronous configurations, and so multiple clients can be run concurrently.

this is not entirely correct. maybe we should remove it for now.

**Review by farzadpanahi — CHANGES_REQUESTED** [review]

**Review by farzadpanahi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (josiah-tesfu) [reply README.md]: Made the change, thanks!

**Review by naomiven — APPROVED** [review]

**Pull request merged**